spec/features/auth/login_feature_spec.rb

```diff
@@ -34,6 +34,27 @@
     expect(page).to_not have_content("The first user to be created will have admin permissions !")
   end
 
+  scenario "Skips validation of minimum password length when authenticating via LDAP" do
+    APP_CONFIG["ldap"] = { "enabled" => true }
+
+    # Skipping validation for LDAP users is configured when the user model is first interpreted
+    # Use a clean room to guard against side effects
+    module CleanRoom
+      eval File.read(File.join(Rails.root, "app", "models", "user.rb"))
```

Comment:
@tboerger made me think. It should be possible to avoid `eval` by doing the following no ?

```ruby
module CleanRoom
  User = ::User
end
```